extensions/amp-analytics/0.1/vendors/blackcrowai.json

```diff
@@ -0,0 +1,24 @@
+{
+  "requests": {
+    "pageview": "https://api.blackcrow.ai/v1/js/events/view"
+  },
+  "triggers": {
+    "trackPageview": {
+      "on": "visible",
+      "request": "pageview"
+    }
+  },
+  "transport": {
+    "useBody": true
+  },
+  "extraUrlParams": {
+    "page_referrer_url": "DOCUMENT_REFERRER",
+    "visitor_id": "CLIENT_ID(_ga)",
+    "page_url": "SOURCE_URL",
+    "google_tracking_id": "${google_tracking_id}",
+    "device_info": "USER_AGENT",
+    "site_name": "${site_name}",
+    "page_id": "home",
```

Comment:
is page_id customizable? can this be included in the request instead? Your call

Reply:
No, it is a static value important for us to understand this traffic is coming from AMP pages. I did update the value though, thanks! 👍